parsers/ENTSOE.py

```diff
@@ -455,8 +455,12 @@ class WindAndSolarProductionForecastTypes(Enum):
         "expected_range": (2000, 25000),
     },
     "RS": {
-        "required": ["coal"],
+        "required": ["biomass", "coal", "gas", "hydro", "unknown"],
         "expected_range": {
+            "coal": (
+                800,
```

Comment:
I think 800 is a safe lower limit based on our historical data. Together with the other required modes it should cover most of the cases.